daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.wf.json

```diff
@@ -9,8 +9,8 @@
       "Description": "Build datestamp used to version the image."
     },
     "google_cloud_repo": {
-      "Value": "DEPRECATED",
-      "Description": "DEPRECATED. Included to ease transition of our pipelines, but not used."
+      "Value": "stable",
+      "Description": "The Google Cloud Repo branch to use."
```

Comment:
Shouldn't this still be DEPRECATED?

Reply:
Set this back to DEPRECATED, although the other rocky EL builds i looked at still show Stable. Might be that the EL builds needed to be updated.